.github/CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+### mondoohq/docs CODEOWNERS
+
+# All resource updates require editorial review by Docs Editor Letha
+*.lr letha@mondoo.com
```

Comment:
We need to define a deputy in case Letha is not available

Reply:
I can do that if she's not around